libs/platform/core/src/services/token-resolver/token-resolver.service.ts

```diff
@@ -7,14 +8,26 @@ export type ResolvedToken = Observable<
 >;
 export type ResolvedResult = DirectiveResult | string | boolean | null;
 
-export type Resolver = () => Observable<ResolvedResult>;
+export type Resolver = (
+  options?: TokenResolverOptions
+) => Observable<ResolvedResult>;
 
 export interface TokenResolver {
-  resolveToken(token: string): Observable<ResolvedResult>;
+  resolveToken(
+    token: string,
```

Comment:
again, looks like better approach is to go with object instead of multiple parameters